core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -1247,6 +1247,16 @@ path_get(wasm_exec_env_t exec_env, struct fd_table *curfds,
     if (path == NULL)
         return convert_errno(errno);
 
+#ifdef BH_PLATFORM_WINDOWS
```

Comment:
Not strictly related to this PR but I realized we'll need it on Windows so paths with backslashes are expanded properly. It would probably be good to verify the behaviour of backslashes with an additional WASI test.